debezium-quarkus/extensions/pom.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.debezium.quarkus</groupId>
+        <artifactId>debezium-quarkus-parent</artifactId>
+        <version>1.1.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>debezium-quarkus-extensions</artifactId>
```

Comment:
Is this intermediary POM really needed? Or could it also be removed?

Reply:
We can likely remove it and reintroduce it later if we add more extensions.